packages/framework/tree-agent/README.md

```diff
@@ -0,0 +1,5 @@
+# @fluidframework/tree-agent
+
+## Description
+
+An AI library compatible with SharedTree
```

Comment:
We should probably add more detail :)

Also, would recommend grabbing the pragmas for the relevant auto-generated contents from one of our other library packages.